package.json

```diff
@@ -10,7 +10,7 @@
     "lint": "standard --fix",
     "build": "NODE_ENV=production node scripts/build.js",
     "serve": "wds --node-resolve --open --watch",
-    "dev": "concurrently \"npm run build -- --watch\" \"wait-on bundle.js && npm run serve\""
+    "dev": "rm bundle.js && concurrently \"node scripts/build.js --watch\" \"wait-on bundle.js && npm run serve\""
```

Comment:
`rm bundle.js` added so that dev server starts with the most updated bundle

Reply:
Will this run on Windows?

Reply:
heh nope, switched to [rimraf](https://www.npmjs.com/package/rimraf)